popup/popup.css

```diff
@@ -1,5 +1,7 @@
 html, body {
   height: min-content;
+  overflow-y: auto;
```

Comment:
these should be applied only when the search container is shown e.g. it would add a class on `body` so a CSS rule for `body.search-results-shown` would be used